services/app-api/serverless.yml

```diff
@@ -73,6 +73,10 @@ custom:
     main: aws_iam
     val: aws_iam
     production: aws_iam
+  scripts:
```

Comment:
I'm not in favor of invoking this on deploy—should we test it to make sure it doesn't break anything first?

Reply:
@oweingrod-coforma @tblackwe We didn't discuss this in sync. Should we have a separate chat about it?

Reply:
we discussed in sync and will be invoking manually.